gradle/app.versions.toml

```diff
@@ -2,5 +2,5 @@
 minSdk = '24'
 targetSdk = '33'
 compileSdk = '33'
-versionName = '1.41'
-versionCode = '233'
\ No newline at end of file
+versionName = '1.42'
```

Comment:
Hm, strange why we have here changes from the current release branch `release/1.42`

Reply:
I accidentally checked out this branch from release branch